<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SANS Reduction Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QScrollArea" name="scrollArea">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>628</width>
        <height>837</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ThetaDependent</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="thetaDependent">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>SensitivityMaps</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="FileFinderWidget" name="sensitivityMaps" native="true">
         <property name="label" stdset="0">
          <string/>
         </property>
         <property name="optional" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>DefaultMaskFile</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>NormaliseBy</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="normaliseBy"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>SampleThickness</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="sampleThickness"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>BeamRadius</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="beamRadius"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>WaterCrossSection</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QDoubleSpinBox" name="waterCrossSection"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>OutputType</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="outputType"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>CalculateResolution</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="calculateResolution"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>DefaultQBinning</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="defaultQBinning"/>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>OutputBinning</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLineEdit" name="outputBinning"/>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>NPixelDivision</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QSpinBox" name="nPixelDivision"/>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>NumberOfWedges</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QSpinBox" name="numberOfWedges"/>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>WedgeAngle</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QDoubleSpinBox" name="wedgeAngle"/>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>WedgeOffset</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QDoubleSpinBox" name="wedgeOffset"/>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>AsymmetricWedges</string>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QCheckBox" name="asymmetricWedges">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>MaxQxy</string>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <widget class="QDoubleSpinBox" name="maxQxy"/>
       </item>
       <item row="18" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>DeltaQ</string>
         </property>
        </widget>
       </item>
       <item row="18" column="1">
        <widget class="QDoubleSpinBox" name="deltaQ"/>
       </item>
       <item row="19" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>IQxQyLogBinning</string>
         </property>
        </widget>
       </item>
       <item row="19" column="1">
        <widget class="QCheckBox" name="iQxQyLogBinning">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="20" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>PanelOutputWorkspaces</string>
         </property>
        </widget>
       </item>
       <item row="20" column="1">
        <widget class="FileFinderWidget" name="panelOutputWorkspaces" native="true">
         <property name="label" stdset="0">
          <string/>
         </property>
         <property name="optional" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="FileFinderWidget" name="defaultMaskFile" native="true">
         <property name="label" stdset="0">
          <string/>
         </property>
         <property name="optional" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>BinningFactor</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QSpinBox" name="binningFactor"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="applyButton">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileFinderWidget</class>
   <extends>QWidget</extends>
   <header location="global">mantidqt.widgets.filefinderwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
